test/render/test-cases.js

```diff
@@ -779,28 +777,6 @@ export const TEST_CASES = [
     ],
     goldenImage: './test/render/golden-images/grid-lnglat.png'
   },
-  {
```

Comment:
Why is this one deleted? We need tests for both GPU aggregation and CPU aggregation.

Reply:
GPU aggregation is covered in the `screengrid-lnglat-colorRange` test case as default aggregation happens on GPU.

Reply:
There was an extra test case as it had `minColor` and `maxColor` props.